ts/Core/Tooltip.ts

```diff
@@ -85,122 +85,6 @@ declare module '../Core/TooltipOptions'{
-            public destroy(): void;
-            public getAnchor(
-                points: (Point|Array<Point>),
-                mouseEvent?: PointerEvent
-            ): Array<number>;
-            public getDateFormat(
-                range: number,
-                date: number,
-                startOfWeek: number,
-                dateTimeLabelFormats: Record<string, string>
-            ): string;
-            public getLabel(): SVGElement;
-            public getPosition(
-                boxWidth: number,
-                boxHeight: number,
-                point: Point
-            ): PositionObject;
-            public getXDateFormat(
-                point: Point,
-                options: TooltipOptions,
-                xAxis: Axis
-            ): string;
-            public hide(delay?: number): void;
-            public init(chart: Chart, options: TooltipOptions): void;
-            public isStickyOnContact(): boolean;
-            public move(
-                x: number,
-                y: number,
-                anchorX: number,
-                anchorY: number
-            ): void;
-            public refresh(
-                pointOrPoints: (Point|Array<Point>),
-                mouseEvent?: PointerEvent
-            ): void;
-            public renderSplit(
-                labels: (string|Array<(boolean|string)>),
-                points: Array<Point>
-            ): void;
-            public styledModeFormat(formatString: string): string;
-            public tooltipFooterHeaderFormatter(
-                labelConfig: Point.PointLabelObject,
-                isFooter?: boolean
-            ): string;
-            public update(options: TooltipOptions): void;
-            public updatePosition(point: Point): void;
-        }
-        interface TooltipFormatterCallbackFunction {
-            (
-                this: TooltipFormatterContextObject,
-                tooltip: Tooltip
-            ): (false|string|Array<string>);
-        }
-        interface TooltipFormatterContextObject {
-            color: ColorType;
-            colorIndex?: number;
-            key: number;
-            percentage?: number;
-            point: Point;
-            points?: Array<Highcharts.TooltipFormatterContextObject>;
-            series: Series;
-            total?: number;
-            x: number;
-            y: number;
-        }
-        interface TooltipPositionerCallbackFunction {
-            (
-                this: Tooltip,
-                labelWidth: number,
-                labelHeight: number,
-                point: (Point|TooltipPositionerPointObject)
-            ): PositionObject;
-        }
-        interface TooltipPositionerPointObject {
-            isHeader: true;
-            plotX: number;
-            plotY: number;
-        }
-        type TooltipShapeValue = ('callout'|'circle'|'square'|'rect');
-    }
-}
-
 /**
  * Callback function to format the text of the tooltip from scratch.
  *
```

Comment:
The API Declarations should be moved below the default export, followed by an `('');`, to reduce distraction.